pkgs/unified_analytics/lib/src/event.dart

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'enums.dart';
+
+class Event {
```

Comment:
This class should be marked `final` so that other libraries can't extend it.

Reply:
Good call